Cargo.toml

```diff
@@ -8,12 +8,13 @@ license = "MIT OR Apache-2.0"
 name = "cortex-m"
 readme = "README.md"
 repository = "https://github.com/japaric/cortex-m"
```

Comment:
Care to change this while you are at it?

Reply:
Done